charts/celestia-node/templates/statefulset.yaml

```diff
@@ -326,6 +338,14 @@ spec:
                 path: OAZHALLLMV4Q
               - key: my_celes_key_info
                 path: my_celes_key.info
+        - name: tls-certs
+          secret:
+            secretName: tls-da-wss
```

Comment:
Make the secret name configurable via the values.
It should be optional to use.

Reply:
Also, document what the secret should contain.